serverlessworkflow/modules/ROOT/pages/job-services/quarkus-extensions.adoc

```diff
@@ -0,0 +1,245 @@
+[#job-service-quarkus-extensions]
+= Job Service Quarkus Extensions
+:compat-mode!:
+// Metadata:
+:description: Job Service Quarkus extensions in {product_name}
+:keywords: sonataflow, workflow, serverless, job service, quarkus extensions
+// links
+:jobs_service_image_url: https://quay.io/repository/kiegroup/kogito-jobs-service-allinone
+:jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images#jobs-services-all-in-one
+:knative_eventing_url: https://knative.dev/docs/eventing/
+:knative_eventing_trigger_url: https://knative.dev/docs/eventing/triggers/
+:knative_eventing_sink_binding_url: https://knative.dev/docs/eventing/sinks/#sink-parameter-example
+:quarkus_infinispan_client_reference_url: https://quarkus.io/guides/infinispan-client-reference
+:quarkus_kafka_url: https://quarkus.io/guides/kafka
+:quarkus_config_reference_url: https://quarkus.io/guides/config-reference
```

Comment:
I think those could be defined into the `antora.yaml` file instead of repeating in each adoc file

Reply:
hmmm, these are not used on this document.